src/HospitalModel.H

```diff
@@ -123,6 +123,12 @@ void HospitalModel<PCType, PTDType, PType>::treatAgents (PCType& a_agents, /*!<
                 ParallelForRNG(np, [=] AMREX_GPU_DEVICE (int i, RandomEngine const& engine) noexcept {
                     if (!inHospital(i, ptd)) {
                         // agent is not in hospital
+                        // determine if agent dies anyway.
+                        if (Random(engine) < disease_parm_d->m_unhospToDeath[age_group_ptr[i]]) {
+                            is_alive_ptr[i] = 0;
+                            flag_status_ptr[i] *= -1;
```

Comment:
This line need not be here. I don't think it's doing anything (multiplying `0` by `-1`) but is misleading (`flag_status_ptr` is used for tracking hospitalized/ICU/vent agents who recover or die).